config/initializers/teaspoon.rb

```diff
@@ -0,0 +1,7 @@
+unless Rails.env.production?
+  module Teaspoon
+    class SuiteController < ActionController::Base
```

Comment:
because we're monkeypatching and re-opening an existing class, we don't want to `< ActionController::Base` here. If that's causing an error because it's trying to be defined from two separate classes, we should force the `Teaspoon::SuiteController` to load with an explicit require before this:

```ruby
unless Rails.env.production?
  require 'teaspoon/suite_controller'
  # ...
end
```